tsconfig.json

```diff
@@ -0,0 +1,33 @@
+{
+    "compilerOptions": {                       
+      "module": "esnext",     
+      "plugins": [
+        {
+          "name": "ts-lit-plugin"
+        }
+      ],       
+      "target": "ES6",         
+      "strict": true,                             
+      "noImplicitAny": false,                      
+      "esModuleInterop": true,                    
+      "skipLibCheck": true,                        
+      "outDir": "./static/dist",                         
```

Comment:
I keep the ourDir to avoid javascript files getting overwritten as stated 
> When working with .d.ts files for JavaScript files you may want to use [emitDeclarationOnly](https://www.typescriptlang.org/tsconfig/#emitDeclarationOnly) or use [outDir](https://www.typescriptlang.org/tsconfig/#outDir) to ensure that the JavaScript files are not overwritten. [https://www.typescriptlang.org/tsconfig/#emitDeclarationOnly](url)